tests/integration/test_reload.py

```diff
@@ -146,3 +146,71 @@ async def test_reload_failure(
 
         # the config should be unchanged
         assert schd.config.cfg['scheduling']['graph']['R1'] == 'one'
+
+
+async def test_orphan_reload(
+    flow,
+    scheduler,
+    start,
+    log_filter,
+):
+    """Reload should not fail about orphaned tasks.
+
+    The following aspects of reload about orphans are tested:
```

Comment:
Can "reload about orphans" be rephrased more clearly? "reload after removing tasks from the workflow config" maybe?